clang/test/Preprocessor/embed_codegen.cpp

```diff
@@ -43,8 +43,9 @@ a
 };
 
 // CHECK: store i32 107, ptr %b, align 4
-int b =
+int b = (
 #embed<jk.txt>
+    )
```

Comment:
Oh nice, these changes fixed a bug -- this code should not have compiled before.

Reply:
In fact, they fix several bugs.